src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ComService.kt

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service
+
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.CommunityOffenderManager
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.UpdateComRequest
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.repository.CommunityOffenderManagerRepository
+
+@Service
+class ComService(
+  private val communityOffenderManagerRepository: CommunityOffenderManagerRepository,
+) {
+
+  @Transactional
+  fun updateComDetails(comDetails: UpdateComRequest): CommunityOffenderManager {
+    val com = this.communityOffenderManagerRepository.findByStaffIdentifier(comDetails.staffIdentifier)
+    val updatedCom = if (com !== null) {
+      com.copy(email = comDetails.staffEmail)
+    } else {
```

Comment:
Is email the only attribute that can change for a COM?

Reply:
Username and staffId cannot change.. and email is the only thing we store which may vary